docs/Configuration.md

```diff
@@ -136,6 +137,139 @@ categoriesConfig: {
 For each category you have to define its processing type (`streaming` in examples above). You can read about processing
 types and their configurations below.
 
+### Process Toolbar Configuration
+
+Toolbars and buttons at process window are configurable, you can provide your own default process toolbar configuration by:
+
+```
+processToolbarConfig {
+  defaultConfig {
+    topLeft: [
+      { type: "tips-panel" }
+    ]
+    topRight: [
+      {
+        type: "process-info-panel"
+        buttons: [
+          { type: "process-save", disabled: { subprocess: false, archived: true, type: "oneof" } }
+          { type: "custom-link", title: "Metrics for {{processName}}", templateHref="/metrics/{{processId}}" }
```

Comment:
Can we describe what placeholders are possible (processName/processId/...)? 
Let's write:
- that you can add own toolbars
- that there are some predefined panels/buttons (check defaults for detailed list...)

Reply:
Done